LetsDoStuff.Domain/LetsDoStuff.Domain.csproj

```diff
@@ -11,12 +11,14 @@
   </PropertyGroup>
 
```

Comment:
Please, update to .net 5.

Reply:
Are you sure it is necessary for the task?